README.md

```diff
@@ -1,75 +1,137 @@
-## Django Framework Skeleton for Google App Engine
+## Python Django Skeleton for Google App Engine
 
 A skeleton for building Python applications on Google App Engine with the
-[Django framework](https://www.djangoproject.com/), using [djangoappengine](http://djangoappengine.readthedocs.org/en/latest/).
-
-This skeleton is a bare-bones "hello world" application.
+[Django Web Framework](https://www.djangoproject.com/) using 
+[Cloud SQL](https://cloud.google.com/sql/docs/introduction).
 
 See our other [Google Cloud Platform github
 repos](https://github.com/GoogleCloudPlatform) for sample applications and
-scaffolding for other Python frameworks and use cases.
+scaffolding for other python frameworks and use cases.
+
+Note that this project no longer uses the built-in Django library but
```

Comment:
Replace `the built-in Django library` with `the SDK-provided Django 1.5 library`